cli/UploadSourcemaps.ts

```diff
@@ -57,7 +57,7 @@ const uploadSourcemaps = async (opts: UploadSourcemapsOptions) => {
   const version = {
     code: opts.code,
     name: opts.name,
-    codePush: opts.label,
+    codepush: opts.label,
```

Comment:
I think we should keep it as `codePush` since the name on [their repo](https://github.com/microsoft/code-push) is in Pascal case (CodePush).